src/sentry/static/sentry/app/options.jsx

```diff
@@ -1,36 +1,97 @@
 import React from 'react';
+import _ from 'underscore';
 import ConfigStore from './stores/configStore';
 import {t} from './locale';
-import {EmailField, TextField} from './components/forms';
+import {EmailField, TextField, BooleanField} from './components/forms';
 
-const definitions = {
-  'system.url-prefix': {
+// This are ordered based on their display order visually
+const sections = [
+  {
+    key: 'system',
+  },
+  {
+    key: 'mail',
+    heading: t('Outbound email'),
+  }
+];
+
+// This are ordered based on their display order visually
+const definitions = [
+  {
+    key: 'system.url-prefix',
     label: t('Root URL'),
     placeholder: 'https://sentry.example.com',
     help: t('The root web address which is used to communicate with the Sentry backend.'),
     defaultValue: () => `${document.location.protocol}//${document.location.host}`,
   },
-  'system.admin-email': {
+  {
+    key: 'system.admin-email',
     label: t('Admin Email'),
     placeholder: 'admin@example.com',
     help: t('The technical contact for this Sentry installation.'),
-    // TODO(dcramer): this shoudl not be hardcoded to a component
+    // TODO(dcramer): this should not be hardcoded to a component
     component: EmailField,
     defaultValue: () => ConfigStore.get('user').email,
   },
-  'system.rate-limit': {
+  {
+    key: 'system.rate-limit',
     label: t('Rate Limit'),
     placeholder: 'e.g. 500',
     help: t('The maximum number of events the system should accept per minute. A value of 0 will disable the default rate limit.'),
   },
-};
+  {
+    key: 'mail.from',
+    label: t('Email From'),
+    component: EmailField,
+    defaultValue: () => `sentry@${document.location.hostname}`,
```

Comment:
I'm open to a different option here. I am trying to present something more useful rather than `root@localhost`. So this should give the user a more sane default. Having `root@localhost` also triggers email to basically go right into spam in my testing.